### ISSUE TITLE
Question about using the model on other datasets 

### ISSUE THREAD

**Issue by parsadlr** [SHI-Labs/SGL-Retinal-Vessel-Segmentation#6]
Hi, 
Great research and thank you for sharing it here. 
I was wondering how can I use your pre-trained models to segment retinal vasculature of fundus images from a dataset of my interest. I don't have any ground-truth labels for it, and I just need to perform some tests on vessels maps.

**Comment by yzhouas** [comment]
Hi, thanks for your interests.
We currently do not have a demo code, but for testing on your own image, you can try formatting your dataset in the form of either CHASE or DRIVE (say image size and folder organization), then you can run the run_test.sh script using one of the models we trained. 

We are not sure about the generalization ability if you have not test the model within the same dataset. Most of the training data in this area have strong domain issues, so it will be common if the testing performance is not good enough. One possible way to improve it is to train the model jointly with your data and the data with ground truth. You can try some self-supervised learning experiments like that.
